src/main.rs

```diff
@@ -7,6 +7,9 @@
 extern crate fail;
 #[macro_use]
 extern crate prometheus;
+#[cfg(test)]
+#[macro_use]
+extern crate serde_json;
```

Comment:
Note `extern crate` isn't necessary anymore in Rust 2018, and also macros can be imported just like other items (functions/structs/etc).